cspell.json

```diff
@@ -88,7 +88,23 @@
     "Ildebrando",
     "Quinchoa",
     "Osuna",
-    "jhernandez"
+    "jhernandez",
+    "entorno",
+    "ejecución",
+    "permite",
+    "entorno",
+    "ejecución",
+    "permite",
+    "usar",
+    "lenguaje",
+    "fuera",
+    "navegador",
+    "caso",
+    "usamos",
+    "configurar",
+    "ejecutar",
+    "servidor",
+    "desarrollo"
```

Comment:
Why are this words in spanish here? Are used on some place?

Reply:
Was used, but currently no, was a error

Reply:
Can u remove it, please?